docs/cli/pyats_develop.rst

```diff
@@ -0,0 +1,129 @@
+pyats develop
+=============
+
+The process of cloning pyATS repositories and putting pyATS packages into 
+development mode can be repetitive and time consuming, especially when working 
+with many packages. This is where the `pyats develop` command comes in. This 
+command is your one-stop-shop for not only cloning pyATS repositories, but for 
+putting them into development mode too. List the packages you want, set the 
+options you'd like, (maybe go make yourself a cup of tea/coffee/cocoa) and let 
+`pyats develop` handle the rest. 
+
+The `pyats develop` helper command is designed to ease and expedite the process 
+of cloning pyATS package repositories and putting each of those packages into 
+development mode. It checks that all of your pyATS packages are up-to-date, 
+downloads packages (if necessary), puts packages into development 
+mode, and then another check that packages in development mode are up-to-date.
+
+.. note::
+
+  This commands requires Github SSH keys to be set up if internal Cisco 
+  repositories are to be cloned.
+
+.. code-block:: text
+
+    Usage:
+      pyats develop [packages...] [options]
+    
+    Usage Examples:
+      pyats develop all
+      pyats develop genie.libs.sdk --skip-version-check
+      pyats develop genie.libs.parser genie.trafficgen --external
+      pyats develop unicon.plugins genie.libs --delete-repos --directory my_repos
+      pyats develop pyats.config --clone-only
+
+    Description:
+      Puts listed pyATS packages into development mode. Listed packages will have 
+      their repositories downloaded from Github if required and 'make develop' will be 
+      run for each package. By default, internal Cisco repos will be cloned if the 
+      pyATS installation is internal, otherwise external repos will be cloned instead. 
+      Github SSH keys are required to clone internal Cisco packages.
+
+    Develop Options:
+      packages              Packages to put into development mode. Available choices: all, cisco-
+                            distutils, genie, genie.libs, genie.libs.clean, genie.libs.conf,
+                            genie.libs.filetransferutils, genie.libs.health, genie.libs.ops,
+                            genie.libs.robot, genie.libs.sdk, genie.libs.parser, genie.telemetry,
+                            genie.trafficgen, pyats, pyats.contrib, rest.connector, unicon,
+                            unicon.plugins
```

Comment:
Any way to put these into a column for both the docs and the `pyats develop --help` command?

IE
```
package            Packages to put into development mode. Available choices:
                       all
                       cisco-distutils
                       genie
                       ...
```
I feel like it'd be easier to read but I could be wrong

Reply:
It would be and I tried that, but the funny part is that it doesn't do that. Seems like it strips away newlines/carriage returns

![image](https://user-images.githubusercontent.com/33130026/129369087-d7f76b35-2f13-4128-bfa4-aec7c502f564.png)